packages/geo-diary/src/components/pages/appointment/appointment.tsx

```diff
@@ -127,6 +139,18 @@ export const handleGetVendors = (
   }
 }
 
+export const handleGetLandlords = (
+  landlords: LandlordsQueryData | undefined,
+  setAppState: Dispatch<SetStateAction<AppState>>,
+) => () => {
```

Comment:
would you mind making the return type explicit? Difficult to tell what the return is without IDE